mpp/mpp_memutils.F90

```diff
@@ -20,8 +20,6 @@
 !> Routines to initialize and report on memory usage during the model run.
 module mpp_memutils_mod
 
-#include "../include/fms_platform.h"
-
```

Comment:
does this need `use platform_mod`?

Reply:
There are no portable kinds defined from platform in this file, but I've added platform_mod in case there's a dependency in another file that uses mpp_memutils.

Reply:
Because the portable kinds are not made public within mpp_memutils.F90, their inclusion here will not satisfy any downstream dependencies.